File: 30DayHackerRankChallengeJava/src/Day13.java
import java.util.*;

abstract class Book {
    String title;
    String author;

    Book(String title, String author) {
        this.title = title;
        this.author = author;
    }

    abstract void display();
}

// Declare your class here. Do not use the 'public' access modifier.
// Declare the price instance variable
class Day13 extends Book {

    int price;

    Day13(String title, String author, int price) {
        super(title, author);
        this.price = price;
    }


    /**
     *   Class Constructor
     *
     *   @param title The book's title.
     *   @param author The book's author.
     *   @param price The book's price.
     **/
    // Write your constructor here

    /**
     * Method Name: display
     * <p>
     * Print the title, author, and price in the specified format.
     **/
    void display() {
        System.out.println("Title: " + title);
        System.out.println("Author: " + author);
        System.out.println("Price: " + price);
    }
    // Write your method here

// End class
}

class Solution13 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String title = scanner.nextLine();
        String author = scanner.nextLine();
        int price = scanner.nextInt();
        scanner.close();

        Book book = new Day13(title, author, price);
        book.display();
    }
}

File: 30DayHackerRankChallengePython/src/Day13.py
from abc import ABCMeta, abstractmethod


class Book(object, metaclass=ABCMeta):
    def __init__(self, title, author):
        self.title = title
        self.author = author

    @abstractmethod
    def display(self): pass


# Write MyBook class
class Day13(Book):
    price = 0

    def __init__(self, title, author, price):
        super(Day13, self).__init__(title, author)
        self.price = price

    def display(self):
        print("Title: " + title)
        print("Author: " + author)
        print("Price: " + str(price))


title = input()
author = input()
price = int(input())
new_novel = Day13(title, author, price)
new_novel.display()
